The global zIndex still does  not take effect if it is imported on demand  with  use setConfig

The global zIndex still does  not take effect if it is imported on demand  with  use setConfig

import { setConfig } from 'element-plus/lib/utils/config.js'

const install = (app): void => {

app.config.globalProperties.$ELEMENT =  { size: 'small', zIndex: 5000 }

setConfig( { size: 'small', zIndex: 5000 })


components.forEach(component => {
app.component(component.name, component)
})
plugins.forEach(plugin => {
app.use(plugin)
})
}

let ExportData =  { install }

const app = createApp( Entry )

app.use(ExportData)

I still encounter **a new construction problem** when using **the latest version**
 
ERROR  Failed to compile with 1 error                                  

 error  in ./node_modules/element-plus/lib/utils/validators.js

Module not found: Error: Can't resolve '@element-plus/utils/util' in ['/Users/hello/node_modules/element-plus/lib/utils']([url](url))

> I still encounter **a new construction problem** when using **the latest version**
> 
> ERROR Failed to compile with 1 error
> 
> error in ./node_modules/element-plus/lib/utils/validators.js
> 
> Module not found: Error: Can't resolve '@element-plus/utils/util' in ['/Users/hello/node_modules/element-plus/lib/utils'](%5Burl%5D(url))

I am so sorry that I missed this file while fixing the previous one. Will dispatch a hotfix soon 🙏🏻 sorry again

--- a/packages/utils/validators.ts
+++ b/packages/utils/validators.ts
@@ -1,4 +1,4 @@
-import { isNumber } from '@element-plus/utils/util'
+import { isNumber } from './util'
 
 export const isValidWidthUnit = (val: string | number): boolean =>{
   if(isNumber(val)){

